test/e2e/images_test.go

```diff
@@ -204,7 +204,7 @@ WORKDIR /test
 		result := podmanTest.Podman([]string{"image", "list", "-q", "-f", "after=quay.io/libpod/alpine:latest"})
 		result.WaitWithDefaultTimeout()
 		Expect(result).Should(Exit(0))
-		Expect(result.OutputToStringArray()).Should(HaveLen(7), "list filter output: %q", result.OutputToString())
```

Comment:
I am always leery about bumping expected results for a test that used to work.  Are we sure that there's not a problem elsewhere?

Reply:
Yes, @baude rebuild images over the weekend since Quay was complaining.